client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -87,38 +82,6 @@ class EditorSharingPublicizeOptions extends React.Component {
 		recordEvent( 'Opened Create New Sharing Service Dialog' );
 	};
 
-	jetpackModulePopup = () => {
-		let href;
-
-		if ( ! this.props.site || ! this.props.site.jetpack ) {
-			return;
-		}
-
-		href = paths.jetpackModules( this.props.site, 'publicize' );
```

Comment:
This removes the only usage of `paths.jetpackModules`. You can remove it from `lib/paths`.